packages/forms/src/styled/text/StyledTextInput.ts

```diff
@@ -185,11 +184,18 @@ const sizeStyles = props => {
   `;
 };
 
-export const StyledTextInput = styled.input.attrs(props => ({
+export interface IStyledTextInputProps {
```

Comment:
Should `IStyledTextInputProps` extend `ThemeProps<DefaultTheme>`?

Reply:
`ThemeProps` is inferred by the styled-components usage so we don't need to manually include it here.

Reply:
Right, but it needs to be repeatedly and-ed in for the function `props` usage.

Reply:
Correct, but that because those helpers are being used within a `styled-component`. The interfaces are only for the API exposed by the components. This lets us reuse these interfaces across the high-abstaction elements without muddying the prop sheets.

Searching for `& ThemeProps` shows us using this pattern across most of the packages. If we want to readdress how we handle these CSS helpers we might want to do that in a separate PR.